docs/development/windows_support.md

```diff
@@ -128,8 +127,19 @@ These instructions mostly mirror the instructions in the root
 > choco install strawberryperl -y
 > ```
 
-If you prefer to install tools manually, you will need:
+Choco will install for now by default the Python version 3.13. If you want to use older python version
+like 3.11 that is not maintained by the choco, you will need to use choco to uninstall the python version provided by the choco and then
+use some other method to install older python version. For example:
 
+
+> ```bash
+>choco uninstall python
+>winget install --id Python.Python.3.11 -e
+> ```
```

Comment:
As stated before - avoid writing documentation that we know will get outdated. We aren't that sensitive to the Python version, so I would leave mentions of it out entirely, or specify a minimum like 3.11+.

Keep docs focused on what is essential. We don't want to overload users/developers with information that they don't need.